utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -950,7 +950,7 @@ void compare_outputs_pln(Rpp8u* output, Rpp8u* refOutput, RpptDescPtr dstDescPtr
         outputTempRef = refOutput + imageCnt * refOutputSize;
         int height = dstImgSizes[imageCnt].height;
         int width = dstImgSizes[imageCnt].width;
-        int matched_idx = 0;
+        int matchedIdx = 0;
```

Comment:
Need not have the channel loop for PLN1 variant at L957, since now 
PLN3 is decoupled

I think PKD3 and PLN1 can be combined in single function?
Please check

Reply:
Done